test/functional/forall/test-forall-atomic-utils.hpp

```diff
@@ -0,0 +1,80 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_FORALL_ATOMIC_UTILS_HPP__
+#define __TEST_FORALL_ATOMIC_UTILS_HPP__
+
+#include "RAJA/RAJA.hpp"
+
+#include "test-forall-utils.hpp"
+
+using AtomicSeqExecs = camp::list< RAJA::seq_exec,
+                                   RAJA::loop_exec
+                                   //RAJA::simd_exec  // fails for atomic-view with clang9
+                                 >;
+
+using AtomicSeqPols = camp::list<
+#if defined(RAJA_TEST_EXHAUSTIVE)
+                                  RAJA::auto_atomic,
+                                  RAJA::builtin_atomic,
+#endif
+                                  RAJA::seq_atomic
+                                >;
+
+#if defined(RAJA_ENABLE_OPENMP)
+using AtomicOmpExecs = camp::list<
+#if defined(RAJA_TEST_EXHAUSTIVE)
+                                   RAJA::omp_for_nowait_exec,
+                                   RAJA::omp_parallel_for_exec,
+#endif
+                                   //RAJA::omp_parallel_exec<RAJA::seq_exec>, // fails for atomic-basic and atomic-ref-math
+                                   RAJA::omp_for_exec
+                                 >;
+
+using AtomicOmpPols = camp::list<
```

Comment:
Should these follow the naming convention in forall-utils more closely?

Reply:
I think that's a good idea. 👍